CONTRIBUTING.md

```diff
@@ -19,15 +19,12 @@ If you're making a pull request, please ensure it adheres to the following guide
     For example, if one wants to find Python jobs, or to filter for US-only companies, the following descriptions would be useful:
     1. [Foo](foo.io) - We are building new certificate provider based on crypocurrencies. Python, Scala, JS, Full Time
     2. [Bar](bar.io) - Uber for fruit baskets. Node, grunt, JS, workday overlaps New York afternoons (1700 - 2200 UTC)
-13. Any new entry should have meaningful content at the time of the addition to this list
-    — we can't tell whether something is awesome if there is little to see.
-    As a rule of thumb, there should be at least about six months worth of content. 
-14. Companies should have "remote DNA".
+13. Any new entry should have meaningful content at the time of the addition to this list. We can't tell whether something is awesome if there is little to see. As a general rule, there should be at least about six months worth of content.
+14. Companies in this list, if added, should have "remote DNA". This means they are not only remote friendly, but dedicated to remote work as a core of their business.
```

Comment:
I'm sorry I wasn't clear earlier. I meant to suggest using "in this list" to _replace_ "if added" (since the former already implies the addition). The reasoning is that, all things working as intended, the suggested wording would prevent unsuitable content from even starting the addition process, rather than stopping them mid-process (i.e. by a decision of the person who would perform the addition by merging the PR).

Not sure if this clarifies what I meant -- please let me know if not.

Reply:
Makes sense to me! Good point. :)